ee/localserver/zta.go

```diff
@@ -3,12 +3,32 @@ package localserver
 import (
 	"log/slog"
 	"net/http"
+	"strings"
 
 	"github.com/kolide/launcher/pkg/traces"
 )
 
 var (
 	localserverZtaInfoKey = []byte("localserver_zta_info")
+
+	// allowlistedZtaOriginsLookup contains the complete list of origins that are permitted to access the /zta endpoint.
+	allowlistedZtaOriginsLookup = map[string]struct{}{
+		// Release extension
+		"chrome-extension://gejiddohjgogedgjnonbofjigllpkmbf":  {},
+		"chrome-extension://khgocmkkpikpnmmkgmdnfckapcdkgfaf":  {},
+		"chrome-extension://aeblfdkhhhdcdjpifhhbdiojplfjncoa":  {},
+		"chrome-extension://dppgmdbiimibapkepcbdbmkaabgiofem":  {},
+		"moz-extension://dfbae458-fb6f-4614-856e-094108a80852": {},
+		"moz-extension://25fc87fa-4d31-4fee-b5c1-c32a7844c063": {},
+		"moz-extension://d634138d-c276-4fc8-924b-40a0ea21d284": {},
+		// Development and internal builds
+		"chrome-extension://hjlinigoblmkhjejkmbegnoaljkphmgo":  {},
+		"moz-extension://0a75d802-9aed-41e7-8daa-24c067386e82": {},
+		"chrome-extension://hiajhnnfoihkhlmfejoljaokdpgboiea":  {},
+		"chrome-extension://kioanpobaefjdloichnjebbdafiloboa":  {},
+	}
+
+	safariWebExtensionScheme = "safari-web-extension://"
```

Comment:
nit- should this be const instead of var?

Reply:
Ahhh yes, thank you, whoops

Reply:
Updated!